stock_picking_mass_assign/wizard/check_assign_all.py

```diff
@@ -0,0 +1,61 @@
+# © 2014-2016 Camptocamp SA (Guewen Baconnier)
+# © 2014 GRAP (Sylvain Le Gal)
+# © 2017 JARSA Sistemas S.A. de C.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.translate import _
+
+
+class CheckAssignAll(models.TransientModel):
+    _name = 'stock.picking.check.assign.all'
+    _description = 'Delivery Orders Check Availability'
+
+    check_availability = fields.Boolean(
+        'Check Availability',
+        default=True,
+        help="""check this box if you want to check the availability of """
+        """the selected Delivery Orders.""")
```

Comment:
I think you should be use double quote instead triple quote as `"check this box if you want to check the availability of the selected Delivery Orders."`